### DISCUSSION TITLE
CPU x16r

### DISCUSSION THREAD

**Discussion by lukras** [RainbowMiner/RainbowMiner#1752] [Ideas]
Please add the parameter for mining x16r to CpuminerRplant.ps1: [PSCustomObject] @ {MainAlgorithm = "x16r"; Params = ""}

**Comment by RainbowMiner** [comment]
Added. Sorry, I have forgotten to enable it in release v4.7.5.1 - but it's now enabled in the miner module CpuminerJayddee.
